yarn.lock

```diff
@@ -612,6 +617,15 @@
     "@chainsafe/as-sha256" "^0.4.1"
     "@noble/hashes" "^1.3.0"
 
+"@chainsafe/persistent-merkle-tree@^0.8.0":
```

Comment:
we have two different version of this now, I think we wanna avoid that

Reply:
just noticed and pushed the change